pkg/kubelet/apis/cri/runtime/v1alpha3/api.proto

```diff
@@ -1,8 +1,8 @@
 // To regenerate api.pb.go run hack/update-generated-runtime.sh
 syntax = 'proto3';
 
-package runtime.v1alpha2;
-option go_package = "v1alpha2";
+package runtime.v1alpha3;
```

Comment:
@yujuhong i am unable to find @Random-Liu's comment.
Are you suggesting to add this new CRI api while keeping the version `v1alpha2`? 
I think in any case runtimes will have to implement this new API. If we dont bump the version, it might be confusing to see runtimes breaking without any cri version change.

Reply:
I don't think we should bump up the CRI version, because:
1) This is only an alpha feature;
2) GRPC convention is that new field and new method is not breaking change. New client should handle backward compatibility by either handling "unimplemented error", or introduce some kind of feature discovery.

I think for near term, because of 1) we shouldn't bump up the version.
For long term, we'll have better story about feature discovery and versioning story, because of 2), we shouldn't bump up the version.

I'm fine with https://github.com/kubernetes/kubernetes/pull/58973, because it is an API breaking change. It changed an existing field.

But I don't think this change is breaking.

Reply:
I think sticking the the grpc convention on what's considered api breaking (and hence the version bumping) would be good. Let's not bump the version this time.

Given that we have more and more CRI implementations at production quality, we should converge how to manage the never-ending new features :-)  Some feature discovery mechanism would be good for both kubernetes and runtime maintainers/users.

Reply:
reverted the version bump. PTAL